COPYRIGHT

```diff
@@ -1,20 +0,0 @@
-# Copyright (C) 2013  Magenta Aps, http://www.magenta-aps.dk
-#
-# Authors: Carsten Agger (carstena@magenta-aps.dk), Jørgen Ulrik B. Krag
-# (jubk@magenta-aps.dk), and others.
-#
-# This directory contains the BibOS Admin system.
-#
-# The BibOS Admin system  is free software: you can redistribute it
-# and/or modify it under the terms of the GNU General Public License as
-# published by the Free Software Foundation, either version 3 of the
-# License, or (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful, but
-# WITHOUT ANY WARRANTY; without even the implied warranties of
-# MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR
-# PURPOSE.  See the GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-~                                                                 
```

Comment:
I realise it isn't exactly fair to ask you to answer questions about a commit you didn't write, but I don't understand why this is necessary. If we're going to remove this, might we perhaps want to replace it with a proper `AUTHORS` file?

Reply:
That is a fair point you raise. I think the best solution would be to add it as an AUTHORS file instead.

Reply:
OK; in that case, let's merge this for now, and then I'll add that in another PR.